molecule/dynamic/playbook.yml

```diff
@@ -8,11 +8,16 @@
         that: "ansible_version.full is version_compare('2.8', '>=')"
         msg: >
           "You must update Ansible to at least 2.8 to use this version of Pulp 3 Installer."
-
+  tasks:
     - name: Dynamically include prereq roles
       include_role:
         name: "{{ roleinputvar }}"
-      loop: "{{ pulp_roles }}"
+      loop:
```

Comment:
nice!

Reply:
Yeah, this usage of "loop" is an example of replacing the legacy "with_items".